src/Sylius/Bundle/ShopBundle/Resources/views/Common/Macro/breadcrumb.html.twig

```diff
@@ -0,0 +1,31 @@
+{% macro breadcrumb(crumbs = {}, prependHome = true) %}
+    {% from _self import breadcrumb_current_crumb, breadcrumb_crumb %}
+    {% if prependHome %}
+        {% set crumbs = [
+            {
+                url: path('sylius_shop_homepage'),
+                name: 'sylius.ui.home'|trans
+            }
+        ]|merge(crumbs) %}
+    {% endif %}
```

Comment:
This could be perhaps be extracted to twig filter/function? So we will enrich crumbs with homepage, but you would be able to customize it in php

Reply:
WDYT if it comes to something like this (take the idea, not the implementation itself):

```suggestion
{% set crumbs = prepareCrumbs(crumbs, prependHome) %}
```